app/css/views/service.css

```diff
@@ -0,0 +1,6 @@
+.service__edit-tags-link {
+  width: 3rem;
+  height: 3rem;
+  padding-left: 1rem;
+}
```

Comment:
Right, I just use existing approach from recipe view :p We can improve header layout in a separate bug for all views, how does it sound?